packages/crayons-icon/icon-utils/icon-registry.js

```diff
@@ -0,0 +1,13 @@
+export const registerIconLibrary = (items) => {
+  var elements = document.querySelectorAll('fw-icon');
+  items.forEach((item) => {
+    elements.forEach((ele) => {
+      if (item.lib !== 'crayons' || item.lib !== 'system')
+        ele.library === item.lib ? (ele.src = item.path) : false;
```

Comment:
do we need a conditional operator here ?. can we not just use a short circuit operator  ?